website/content/docs/api-gateway/configuration/gateway.mdx

```diff
@@ -183,3 +183,49 @@ tls:
 
 ```
 
+#### Example cross-namespace certificateRef
+
+The following example creates a `Gateway` named `example-gateway` in namespace `gateway-namespace`. This `Gateway` has a `certificateRef` in namespace `secret-namespace`. The reference is allowed because the `ReferenceGrant`, named `reference-grant` in namespace `secret-namespace`, allows `Gateways` in `gateway-namespace` to reference `Secrets` in `secret-namespace`.
+
+<CodeBlockConfig filename="gateway_with_referencegrant.yaml">
```

Comment:
```suggestion
<CodeBlockConfig filename="gateway_with_referencegrant.yaml" lineNumbers=true highlight="2-4","16-18","23-26","30-34">
```
I think this is the correct syntax. So we can make the paragraphing introducing the example a little easier to parse.